src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,102 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  movieList: (newMovie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ movieList }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState<Movie>();
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    if (movie?.Response === 'False') {
+      setError(true);
+    }
+  }, [movie]);
+
+  const getMovie = async (inputTitle: string) => {
```

Comment:
it's better to remove fetch to separate file

Reply:
agree, move fetch to separate file